andrepolischuk/thomas
# Language Selection in Menu - Version 0.6.1

Hello @andrepolischuk ,

I'm excited to introduce a new feature in version 0.6.1 that enhances the user experience of our application. This pull request adds the capability to switch between different languages for the menu, making the application more accessible to a wider audience.

**What's Changed:**

1. **Language Selection:** I've implemented a language selection feature in the menu, allowing users to choose between English, Portuguese (pt-br), Italian, and French languages. This feature enhances the usability of the application for users around the world.

2. **Menu Structure Update:** To accommodate the language selection functionality, I've made careful modifications to the menu structure. Now, users can easily switch between languages by accessing the "Services" submenu in the "Thomas" menu.

3. **Code Enhancement:** In addition to the main feature, I've also taken the opportunity to address minor formatting and styling issues in the code. This ensures a consistent and visually pleasing experience for our users.

4. **Improved Documentation:** I've updated the `menuTemplate.js` file to reflect the changes and added languages. This keeps the documentation up-to-date and helps contributors understand the new feature.

**How to Test:**

1. Clone this branch to your local environment.
2. Launch the application and navigate to the "Thomas" menu.
3. Observe the "Services" submenu, which now includes options to switch between languages.
4. Select your preferred language and see the menu labels change instantly.
5. Test other menu functionalities and ensure that the application works smoothly with the new feature.

I believe that this addition will make our application more inclusive and user-friendly. Your feedback and testing are highly appreciated. Let's collaborate to deliver a great experience to our users!

Note:
I spoke with you some time ago via email, asking for permission to translate Thomas into other languages.

Cheers,
Yuri.
